### PR TITLE
feature(ui): `/myRecentJobs` page completed

### LINKED ISSUE #226: Creation of `my recent jobs` page
### Description

Currently, the `/myRecentJobs` page is not implemented as it gives us a blank response. 

#### Steps followed and expected result
Run the UI with the current backend and after login visit the My recent jobs page

#### Process that we can use to filter the Jobs according to the user
we can use the `users/self` API endpoint in order to get the logged-in user's ID and thus from the jobs list provided by the `/jobs` we can filter the jobs using the `userId:` parameter of each job.

PS: I would like to work on this issue

@shaheemazmalmmd @GMishx

### PR DESCRIPTION
**Pull request by soham4abc**
## Description
`/allRecentJobs` page UI is completed with a table featuring dynamic search and pagination. Also with a customizable number of rows.

### Changes

A new package has been installed MDBreact (https://github.com/mdbootstrap/mdb-react-ui-kit)
(same in PR #223 )

## How to test

install the updated dependencies and run the server to visit the `/myRecentJobs` page

### Screenshots
User 1 perspective:
Screenshot from 2022-06-17 19-27-39

User 2 perspective:
Screenshot from 2022-06-17 19-28-09

Fixes #226 

@shaheemazmalmmd @GMishx @sjha2048 PTAL

### CODE REVIEW

File: src/pages/Jobs/MyRecentJobs/index.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
- Copyright (C) 2021 Shruti Agarwal (mail2shruti.ag@gmail.com)
+ Copyright (C) 2021 Shruti Agarwal (mail2shruti.ag@gmail.com),
+ Soham Banerjee (sohambanerjee4abc@hotmail.com)
```
Comment (GMishx) [inline src/pages/Jobs/MyRecentJobs/index.jsx:3]: Please add copyright year (`Copyright (C) 2022 Soham Banerjee (sohambanerjee4abc@hotmail.com)`)

File: src/pages/Jobs/MyRecentJobs/index.jsx [hunk 2]
```diff
@@ -16,20 +17,114 @@
...
+      },
+    ],
+    rows: jobsDataList,
+  };
+
   return (
     <>
-      <Title title="My Recent Jobs" />
-      <div className="main-container my-3">
+      <Title title="Show Jobs" />
```
Comment (GMishx) [inline src/pages/Jobs/MyRecentJobs/index.jsx:113]: Any reason to change the title here?

**Review by GMishx — CHANGES_REQUESTED** [review]
Along with the changes, I am getting following error while compiling.
Is it some issue from my end?

```
Failed to compile.

src/pages/Jobs/MyRecentJobs/index.jsx
  Line 31:30:  Unable to resolve path to module 'mdbreact'  import/no-unresolved

Search for the keywords to learn more about each error.
```

**Comment by GMishx** [comment]
Also, the commit message is in wrong format. It should be
`feat(ui): my recent jobs page completed`

File: src/pages/Jobs/MyRecentJobs/index.jsx (hunk 1 above)
Comment (soham4abc) [reply src/pages/Jobs/MyRecentJobs/index.jsx:3]: Resolved

File: src/pages/Jobs/MyRecentJobs/index.jsx (hunk 2 above)
Comment (soham4abc) [reply src/pages/Jobs/MyRecentJobs/index.jsx:113]: No particular reason reverting it to the original title!
Resolved

**Comment by soham4abc** [comment]
> Along with the changes, I am getting following error while compiling. Is it some issue from my end?
> 
> ```
> Failed to compile.
> 
> src/pages/Jobs/MyRecentJobs/index.jsx
>   Line 31:30:  Unable to resolve path to module 'mdbreact'  import/no-unresolved
> 
> Search for the keywords to learn more about each error.
> ```

Did you run `yarn install`? A new package is used in this PR. If not please do so and restart the server. If that too doesn't work.. try deleting old `node_modules` and run `yarn install` to get fresh list of packages. 

Thank you, do let me know if the problem persists.

File: src/pages/Jobs/MyRecentJobs/index.jsx (hunk 2 above)
Comment (GMishx) [reply src/pages/Jobs/MyRecentJobs/index.jsx]: I was not asking for a revert :-)
Just wanted to understand the reason behind it.

**Comment by GMishx** [comment]
> 

I did the exact same steps but it failed last time.

I did the exact same steps again and it worked this time. :-P

**Comment by soham4abc** [comment]
> Looks like theming issue image
> 
> image

Yes this is a theming issue

**Comment by soham4abc** [comment]
Fixed @GMishx 

Screenshot from 2022-06-21 14-48-01

**Pull request closed without merging**